src/references/interaction_system.md

```diff
@@ -0,0 +1,154 @@
+
+#### Example
+
+- Hovering the mouse over the ``?`` icon reveals a tooltip using the **Toggle Tooltip Component**.
+- Clicking and dragging on the outer part of the window will reposition it using the **Draggable Object Component**.
+- Each of the white boxes will change color when clicked using the **Change Color Component**, and add themselves to a selection set using the **Selectable Component**.
+
+<!-- TODO: Add video of interaction example -->
+
+### Look
+
+The **Look Interaction Example** checks if a player's reticle (or the center of the screen, regardless if there is a displayed reticle) is intersecting a specific object.
+
+#### Example
+
+- When the center of the screen is over the sphere, it should turn yellow using the **Change Color Component**, and show a ring on the floor using the **Toggle Visibility Component**.
+- When clicked, the sphere should change color to Blue, also from the **Change Color Component**.
+<!-- TODO: Add video of interaction example -->
+
+### Pointer
+
+Similar to the **Cursor Interaction Module**, the **Pointer Interaction Module** uses the position of the cursor, but for objects in the world, rather than UI. It has the added benefit of being able to detect the angle of intersection with the mouse and the surface of the target object.
+
+#### Example
+
+Drag the **Pointer Interaction Template** from **Project Content** into the scene, and start a preview. The cursor will be visible, and the sphere can be interacted by moving the cursor over it, and by clicking.
+
+- Moving the mouse over the cube from any point should change its color using the **Change Color Component**.
+- Clicking the sphere will also change its color using the **Change Color Component**, but also spawns VFX Muzzle Flash that is rotated to show the direction of collision between the pointer and the object, using the **Spawn Object Component** with the **UseInputNormal** and **UseInputPosition** properties enabled.
+
+<!-- TODO: Add video of interaction example -->
+
+### Trigger
+
+The **Trigger Interaction Module** works much like a client-side interactable trigger, but allows for custom keybindings and has more events that can be accessed for that interaction.
+
+!!! info
+    This module will react to Trigger interactions by the local Player or all Players depending on the Input Settings. By default the Trigger module will only react to the local Player in order  to act like the other Input Modules. If the setting "TriggerLocally" is false then Trigger inputs from all Players will run. This can be a useful way to mimic networked play without using any actual networking but it could affect your interactions in unexpected ways.
+
+<!-- TODO: Add video of interaction example -->
+
+#### Example
+
+Drag the **Trigger Interaction Template** from **Project Content** into the scene, and start a preview. You can interact with the cube by walking close enough to overlap its trigger, and then clicking.
```

Comment:
"**You** can interact with the cube"

Should this be third person (i.e. Creators)?